app/partials/app-status.jsx

```diff
@@ -53,12 +53,13 @@ export default class AppStatus extends React.Component {
     })
     .then((text) => {
       console.log('AppStatus: Received status data from ' + APP_STATUS_URL + '.');
-      if (!text || text === '') {
+      const cleanedtext = text.trim();  //If text is just white space or newlines...
```

Comment:
That's a good point - I overlooked that my `null` check was moved after the trim. Update shortly.